cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -22,16 +22,18 @@ module Cardano.Tracing.Tracers
   , nullTracers
   ) where
 
-import           Cardano.Prelude hiding (atomically, show)
+import           Cardano.Prelude hiding (show)
 import           Prelude (String, show)
 
 import           GHC.Clock (getMonotonicTimeNSec)
 import           Control.Tracer
+-- import           Debug.Trace (trace)
 
 import           Codec.CBOR.Read (DeserialiseFailure)
-import           Data.Aeson (ToJSON)
+import           Data.Aeson (ToJSON(..), Value(..))
 import           Data.IORef (IORef, atomicModifyIORef', readIORef)
 import qualified Data.Text as Text
+import qualified Data.HashMap.Strict as Map
```

Comment:
Why not an ordered `Map` like everywhere else?

Reply:
We're creating an Aeson object manually, which uses this HashMap type under the hood.